### ISSUE TITLE
Physics?

### ISSUE THREAD

**Issue by Carrott2016** [Riqorm/PlayHomeVR#2]
Hi, this is great! Happy so see someone continuing Eusth´s work!
Any chance to enable breast physics with the VR controllers at some point like in PlayClub?

Excellent work so far. Thanks a lot!

**Comment by Riqorm** [comment]
Yeah, it's definitely possible - as you may have seen already there is already a bit interaction in the current release, when you pull of cloth which covers the breasts.
Can't promise anything though and to be honest I'm not a huge fan of it itself, as it somewhat lacks interaction. It would be nice if the girl would react to it in some way (and that might be a direction to go for the obey tool), however it would soon hit a roadblock, as any kind of convincing reaction really needs voice acting.

**Comment by Carrott2016** [comment]
Thank you so much for the quick update, Riqorm!

Little feedback: 
Touch works nicely on the main character, but seems not to work initially on the "guest" character. However, touch works on main and guest characters, if you swapped/replaced them one time within the H-scene, so maybe just a issue of missing initialization?

Obey tool does not work at all on guest character.

**Comment by Carrott2016** [comment]
Oh, it seems your mod is getting a bit more attenion now than on HF. Found this on reddit:
https://www.reddit.com/r/adultvrgames/comments/7add3e/heads_up_stay_away_from_this_new_vr_mod/
